src/schemathesis/cli/output/default.py

```diff
@@ -319,6 +319,8 @@ def handle_finished(context: ExecutionContext, event: events.Finished) -> None:
     display_failures(context, event)
     display_application_logs(context, event)
     display_statistic(context, event)
+    if context.junit_xml_file:
```

Comment:
I think it can be in `display_statistic`, in a similar way as cassettes feature does